src/components/ConfirmDialog.react.js

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import {Component} from 'react';
+
+/**
+ * ConfirmDialog wraps window.confirm
```

Comment:
This is the message that will appear in the component's `help(dcc.ConfirmDialog)`, so let's expand this a bit. The Dash users probably don't know what `window.confirm` is referring to. Perhaps something like:
> ConfirmDialog is used to display the browser's native "confirm" modal, with an optional message and two buttons ("OK" and "Cancel"). This `ConfirmDialog` can be used in conjunction with buttons when the user is performing an action that should require an extra step of verification. 

And then similarly for the `ConfirmDialogProvider`. For the `ConfirmDialogProvider`, we should mention that you can pass in a button directly as `children`